src/windows/wslaservice/inc/wslaservice.idl

```diff
@@ -127,6 +247,14 @@ interface IWSLASession : IUnknown
 interface IWSLAUserSession : IUnknown
```

Comment:
I do think we should rename this to `ISessionManager` or similar. Do you prefer to do that later?